src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -17,7 +17,7 @@
 
 class Configuration implements ConfigurationInterface
 {
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('seo_override');
         $rootNode = method_exists(TreeBuilder::class, 'getRootNode') ? $treeBuilder->getRootNode() : $treeBuilder->root('seo_override');
```

Comment:
This check is not needed anymore. In SF 4.4, the method `getRootNode` is always here

https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L61